gplearn/genetic.py

```diff
@@ -479,7 +479,11 @@ def fit(self, X, y, sample_weight=None):
         if isinstance(self, TransformerMixin):
             # Find the best individuals in the final generation
```

Comment:
I don't think this will completely allow all kinds of customized metrics ( e.g. selecting models other than correlation approach), since the content below written in hard code is based on correlation ..